java/cukes/src/test/java/dev/sunbirdrc/registry/test/RegistryTestBase.java

```diff
@@ -27,11 +27,11 @@ public class RegistryTestBase {
 	private static final String CONTEXT_CONSTANT = "teacher:";
 	protected static Type mapType = new TypeToken<Map<String, String>>() {
 	}.getType();
-	private static String ssoUrl = System.getenv("sunbird_sso_url");
-	private static String ssoClientId = System.getenv("sunbird_sso_client_id");
-	private static String ssoUsername = System.getenv("sunbird_sso_username");
-	private static String ssoPassword = System.getenv("sunbird_sso_password");
-	private static String ssoRealm = System.getenv("sunbird_sso_realm");
+	private static String ssoUrl = System.getenv("IDENTITY_URL");
```

Comment:
THese are not Identity services. These are SSO credentials

Reply:
Yes, these are SSO credentials and can be renamed but If we check the file _`java/middleware/registry-middleware/identity-provider/src/main/java/dev/sunbirdrc/registry/identity_providers/pojos/IdentityProviderConfiguration.java`_ we are using `identity.url`, `identity.provider`, ...etc. I just tried to not to change the convention but follow it in other places as well.